src/librustdoc/lib.rs

```diff
@@ -412,10 +412,11 @@ fn opts() -> Vec<RustcOptGroup> {
             )
         }),
         unstable("test-builder", |o| {
-            o.optflag(
+            o.optopt(
                 "",
                 "test-builder",
-                "specified the rustc-like binary to use as the test builder",
+                "specifies the rustc-like binary to use as the test builder",
```

Comment:
I would use the same wording as 'runtool' above.
```suggestion
                "The rustc-like binary to use as the test builder",
```